### ISSUE TITLE
Question 

### ISSUE THREAD

**Issue by ajulian3** [Statistical-Connectomics-Sp15/intro#1]
What are common algorithms that are important in detecting neural connectivity? Specifically, what analytics platforms are they on and how are they optimized?

**Comment by willgray** [comment]
I might be able to help with this...do you mean, how do we estimate graphs from brains?

This is often a computer vision/engineering problem that is modality specific...do you have something in mind?

**Comment by ajulian3** [comment]
Yes. I was curious as to what additional toolbars or packages would be necessary to take into account standardizing the neural network information. I prefer to perform neural network analysis on the R platform. Do you think there is a better option?

**Comment by willgray** [comment]
I see - so basically - what tools are there for preprocessing/standardizing brain graphs?  We often use graphml as a standard, which is supported by MATLAB/R/Python/igraph/lots of stuff.  OCP also supports a conversion service...so given a graph in some format and an analysis tool of your choice, you should be good to go.  Does this answer your question?  Or do you wonder where the graphs come from?

http://openconnecto.me/graph-services/welcome/

**Comment by jovo** [comment]
igraph is the R package we help develop
FlashGraph is one that we develop on our own (for much larger graphs).
GraphX is a python package that people like.

On Tuesday, February 3, 2015, William Gray <notifications@github.com> wrote:

>  I see - so basically - what tools are there for
> preprocessing/standardizing brain graphs? We often use graphml as a
> standard, which is supported by MATLAB/R/Python/igraph/lots of stuff. OCP
> also supports a conversion service...so given a graph in some format and an
> analysis tool of your choice, you should be good to go. Does this answer
> your question? Or do you wonder where the graphs come from?
>
> http://openconnecto.me/graph-services/welcome/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Statistical-Connectomics-Sp15/intro/issues/1#issuecomment-72723658>
> .
>


-- 
the glass is all full:  half water, half air.
openconnecto.me, we're hiring!
<https://docs.google.com/document/d/14SApYAzxF0Ddqg2ZCEwjmz3ht2TDhDmxyZI2ZP82_0U/edit?usp=sharing>
, jovo.me, my calendar
<https://www.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York>

**Comment by jovo** [comment]
also, check out http://thefreemanlab.com/thunder/
